latest/src/home/config/autostart/disable-screensaver.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Type=Application
+Name=disable-screensaver
+Comment=Disable screensaver
+Terminal=true
+Exec=xset s off
```

Comment:
This is the only solution with which I was able to disable auto screen lock.

I also tried the followings but didn't work:
- with `~/.xscreensaver` (didn't work because screensaver is not installed)
- with `~/.xinitrc` (maybe not supported)
- with `~/.Xresources`

Reply:
@fmoc Please, review what you think of this approach.

Reply:
Looks good, using X11 directly may be the best option. I hope it'll work reliably!